cliconfig/config_test.go

```diff
@@ -75,14 +73,105 @@ func TestEmptyJson(t *testing.T) {
 	}
 
 	// Now save it and make sure it shows up in new form
-	err = config.Save()
+	saveConfigAndValidateNewFormat(t, config, tmpHome)
+}
+
+func TestOldInvalidsAuth(t *testing.T) {
+	invalids := map[string]string{
+		`username = test`: "The Auth config file is empty",
+		`username
+password
+email`: "Invalid Auth config file",
+		`username = test
+email`: "Invalid auth configuration file",
+		`username = am9lam9lOmhlbGxv
+email`: "Invalid Auth config file",
+	}
```

Comment:
The actual error messages are a little bit strange, should probably be changed a little bit.